packages/@coorpacademy-components/src/molecule/skills-chart-side-information-item/test/fixtures/default-courses-completed.js

```diff
@@ -0,0 +1,11 @@
+import Icon from '../../../../atom/icon/test/fixtures/default-with-only-icon-name-and-bg-color-green-hex';
+
+const iconProps = Icon.props;
+export default {
+  props: {
+    title: 'Courses completed',
+    value: '165',
+    legend: 'focused skills',
```

Comment:
```suggestion
    legend: 'on <b>focused skills</b>',
```

Reply:
passing html tags like this doesn't work, (its not interpreted), in order to work we have to use a react parser lib like html-react-parser, i beleive i implemented a better soluttion: i split legend into two parts, prefix and main.

Reply:
WE Can use

// eslint-disable-next-line react/no-danger
        dangerouslySetInnerHTML={{__html: legend}}
      />